web-dev/src/components/DetailScreen/comment-section.js

```diff
@@ -0,0 +1,56 @@
+import "./detail-screen.css";
+import CommentItem from "./comment-item";
+import React from "react";
+
+const CommentSection = () => {
+
+  // TODO: retrieve comments from comment API
+  const comments = [
+    {
+      username: "John",
+      critic: true,
```

Comment:
Please confirm with @mnolan34 that the critic field is added to the back end schema. I'm guessing this field should be part of the user schema? Though I think it could also be part of the comments schema. Is the intent that critic comments are shown first/ on top?

Reply:
Sure, I am about to write to him about my API design.